lnmuxrpc/lnmux.proto

```diff
@@ -73,3 +77,23 @@ message CancelInvoiceRequest {
 }
 
 message CancelInvoiceResponse {}
+
+message ListInvoiceRequest {
+    uint64 num_max_invoices = 1;
```

Comment:
Nit: is `max_invoices_count` better? Or something else?

I am currently reading `Number of maximum invoices', which doesn't parse perfectly.